worlds/tunic/items.py

```diff
@@ -48,6 +48,7 @@ class TunicItemData(NamedTuple):
     "Gun": TunicItemData(IC.progression | IC.useful, 1, 30, "Weapons"),
     "Shield": TunicItemData(IC.useful, 1, 31, combat_ic=IC.progression | IC.useful),
     "Dath Stone": TunicItemData(IC.useful, 1, 32),
+    "Torch": TunicItemData(IC.useful, 0, 152),
```

Comment:
```suggestion
    "Torch": TunicItemData(IC.useful, 0, 156),
```
To not overlap with the new money items in breakable shuffle